frontend/packages/ceph-storage-plugin/src/components/dashboards/block-pool/mirroring-card.scss

```
@@ -0,0 +1,29 @@
+.ceph-block-pool-mirroring-card__body {
```

```suggestion
.odf-block-pool__mirroring-card-body {
```
s per BEM you should use element after __

---

Use odf as suffix now

---

ack